### PR TITLE
Feat(frontend): Pair and Mob Programming 236

### LINKED ISSUE #236: Mob Programing Guidelines
[https://docs.google.com/document/d/1Ve5LVAJvGqJbUZR6C2o3ZNvFPKpuJ6sfIwxlpgKpKCk/export?format=pdf](https://docs.google.com/document/d/1Ve5LVAJvGqJbUZR6C2o3ZNvFPKpuJ6sfIwxlpgKpKCk/export?format=pdf)

[https://docs.google.com/document/d/1Ve5LVAJvGqJbUZR6C2o3ZNvFPKpuJ6sfIwxlpgKpKCk/export?format=pdf](https://docs.google.com/document/d/1Ve5LVAJvGqJbUZR6C2o3ZNvFPKpuJ6sfIwxlpgKpKCk/export?format=pdf)

What is mob programming:

Waste in tech debt

Waste in Dev Ops

Group critiques in design - or ideation process

### Mob programming como practica de XP para agregar valor a proyectos, servicios, procesos, organizaciones y roles como recruting y diseño.

**Que es Mob programming?**

Un grupo de personas con mucha inteligencia con grandes retos trabajando en equipo para resolver los retos de manera creativa, con excelencia operacional y trabajando enfocado en entrega de valor constante.

**Como se hace?**

Team size: 3-6 personas

Roles: 

- Conductor: Recibe input del equipo y ejecuta la implementacion de las ideas:
- Navegadores: Son los que le dan direccion al conductor, tambien es la voz del equipo

Bias for action:

- Evitar quedarse en conversaciones abstractas por mucho tiempo
- NO discuta un problema por mas de 10 minutos (Recomendado)
- Si hay mas de una solucion a un problema experimente con dos o 3 (Use paretto)
- Use ejemplos para mantener la conversacion caminando.
- Las ideas de los navegadores solo se implementan por el conductor
- El skill level del conductor es clave para que los navegadores desarrollen mas las instrucciones.

**El valor del mobbing:**

- Soluciones se entregan mas rapido, con mayor calidad por medio del incremento del enfoque
- Mitiga las tareas tediosas, tiene visibilidad 360 y mejora la automatizacion .
- Apalanca el entrenamiento entre miembros del equipo
- Entrega resultados mas rapido reduciendo el tiempo del work in progress y eliminando tiempos de handoff entre equipos y mitiga los problemas de comunicación.
- Puede incrementar el tiempo de entrega de una tarea por la cantidad de personas pensando alrededor de una tarea individual, pero bajan aprox 15% de defectos o margen de error, se mitiga refactoring y retrabajo y soporte de la tarea o el entregable.
- Satisfaccion se incrementa un 96%
- Entrenamiento agil
- Team building and comunication
- Mas opciones de soluciones para varios problemas a la vez.

**Beneficios para otras organizaciones y servicios**

Recruting

Diseño (Service, UX, Production)

- Add pair programming

### PR DESCRIPTION
**Pull request by steph222**
### GH Issue

Resolves #236  

### Steps to test
1. Go to Software Development in Pair or Mob
2. Check content is right.

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is equal  to https://github.com/eoscostarica/guias.eoscostarica.io/pull/216 but in English

### CODE REVIEW

File: docs/pair-mob-programming.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+<div style={{  textAlign: "center" }}>
+<img alt="Image Pair Programing"  src={ useBaseUrl( 'img/PairMobProgramming/PairProgramming.png' )} /> </div>
+
+### How is it done?
+**Team size:** 2 people.
+
+### Roles:
+- **Pilot:** He is charge of writing the code.
+- **Copilot:** He is charge of supervising that code.
```
Comment (xavier506) [inline docs/pair-mob-programming.md:25]: Navigator

File: docs/pair-mob-programming.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+### Roles:
+- **Driver:** Receives input from the team and executes the implementation of ideas.
+- **Navigators:** They are the ones that give the driver direction, it is also the voice of the team.
+
+### Bias for action:
+- Avoid staying in abstract conversations for a long time.
+- Do not discuss a problem for more than 10 minutes (Recommended).
+- If there is more than one solution to a problem, experiment with two or 3 (Use
+paretto).
+- Use examples to keep the conversation going by walking.
```
Comment (xavier506) [inline docs/pair-mob-programming.md:65]: By walking?

File: docs/pair-mob-programming.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+A group of highly intelligent people with great challenges working as a team to solve challenges creatively, with operational excellence and working focused on delivering constant value.
+
+<div style={{  textAlign: "center" }}>
+<img alt="Image Mob Programing" width="70%" src={ useBaseUrl( 'img/PairMobProgramming/MobProgramming.png' )} /> </div>
+
+### How is it done?
+**Team size:** 3-6 people.
+
+### Roles:
+- **Driver:** Receives input from the team and executes the implementation of ideas.
```
Comment (xavier506) [inline docs/pair-mob-programming.md:57]: Also known as pilot

(I prefer driver / navigator?

File: docs/pair-mob-programming.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+Pair programming specifies that there are always two people working on the code at the same time and  as far as possible, they sit together. One is in charge of writing the code and the other of supervising it in real time. At the same time, they are constantly exchanging impressions: they discuss problems, find solutions and develop creative ideas.
+
+<div style={{  textAlign: "center" }}>
+<img alt="Image Pair Programing"  src={ useBaseUrl( 'img/PairMobProgramming/PairProgramming.png' )} /> </div>
+
+### How is it done?
+**Team size:** 2 people.
+
+### Roles:
+- **Pilot:** He is charge of writing the code.
```
Comment (xavier506) [inline docs/pair-mob-programming.md:24]: Driver (also known as pilot)

File: docs/pair-mob-programming.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+- The skill level of the driver is key for navigators to further develop the instructions.
+- Immediate feedback.
+- Learn or contribute.
+- Agility in communication.
+- Retrospective.
+- Empathy and respect.
+
+### Artifacts:
+
+- Backlog of specific problems defined by the product owner or the Tech lead.
```
Comment (xavier506) [inline docs/pair-mob-programming.md:76]: tech lead (no caps)

File: docs/pair-mob-programming.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+- You can increase the delivery time of a task by the number of people thinking about an individual task, but decrease approx 15% of defects or margin of error, refactoring and rework and support of the task or the deliverable are mitigated.
+- Satisfaction increases 96% among programmers.
+- Agile training.
+- Team building and communication.
+- More solution options for several problems at the same time.
+
+### Benefits for other organizations and services.
+
+- Recruiting.
+- Design(Service, UX, Production).
```
Comment (xavier506) [inline docs/pair-mob-programming.md:102]: missing space 

 `Design(Se...`

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by Edgar097 — APPROVED** [review]

**Pull request merged**